docs/manage_infrastructure.md

```diff
@@ -285,7 +285,7 @@ These templates will use PV configuration in order to boot: either from the righ
 
 Because there is already disks installed, you shouldn't have "Install settings" _per se_. But you can use our `config drive` setup if your template already has CloudInit installed!
 
-Please refer to the [XenServer CloudInit section](cloudinit.md) for more.
+Please refer to the [XenServer CloudInit section](https://xen-orchestra.com/docs/advanced.html#cloud-init) for more.
```

Comment:
1. Use internal link: `[CloudInit section](advanced.md#cloud-init)`
2. Change the sentence: XCP-ng instead of XenServer